test/e2e/pods.go

```diff
@@ -689,7 +689,7 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 15,
+							InitialDelaySeconds: 30,
 							FailureThreshold:    1,
```

Comment:
Yeah, just saw your comment on the issue. Lets see if setting the InitialDelaySeconds helps.